multi-agent/tests/unit/resources/conftest.py

```
@@ -0,0 +1,435 @@
+@pytest.fixture
+def builtin_user_config_repo() -> FakeBuiltinUserConfigRepository:
+    return FakeBuiltinUserConfigRepository()
+
+
+@pytest.fixture
+def resource_repo() -> FakeResourceRepository:
+    return FakeResourceRepository()
+
+
+@pytest.fixture
+def resource_registry(resource_repo, element_registry) -> ResourcesRegistry:
```

Ruff (0.16.1)

[warning] Unused function argument: `element_registry`

(ARG001)